personal_assistant/personal_assistant/settings.py

```diff
@@ -46,6 +47,7 @@
     'pa_core',
     'pa_user',
     'pa_contacts', #Додав застосунок для контактів
+    'pa_note'
```

Comment:
Для того, щоб у наступного хто додаватиме свій застосунок, в змінах не відображалося, що змінено два рядки (за необхідності додавати кому) то варто ставити кому навіть після останнього елемента списку.
```suggestion
    'pa_note',
```